### PR TITLE
Configurable zoom limits

### PR DESCRIPTION
**Pull request by mojowen**
Three things:

 * Restores the hover tool tip for quantitative
 * Adds a configuration setting for max zoom, sets it to very large for NY State Senate
 * Adds a prefix `na-` that's used to figure columns [card](https://trello.com/c/4OFn0SwI/27-add-an-option-to-add-ignore-prefix-to-columns-in-sheets-so-we-can-have-calculation-columns-that-dont-get-pulled-in)

### CODE REVIEW

**Review by noahmanger — APPROVED** [review]
Nice!

File: src/map/index.js [hunk 1]
```diff
@@ -116,10 +121,11 @@ const addZoomButtons = zoomFunction => {
     });
 };
 
-const buildZoom = (baseZoom, svgWidth, svgHeight) => {
+const buildZoom = (baseZoom, svgWidth, svgHeight, mapConfig) => {
+  const zoomScaler = mapConfig.zoomScalar || 4;
```
Comment (noahmanger) [inline src/map/index.js]: Small thing but you've got `zoomScaler` and `zoomScalar`.

Comment (noahmanger) [reply src/map/index.js]: Actually, where does `mapConfig.zoomScalar` get set?

Comment (mojowen) [reply src/map/index.js]: good catch - and these are getting set in the map's geojson. At the top level there's a `config` property which will let us change the projection and perform rotations / scale at map init

**Pull request merged**